### ISSUE TITLE
Alarme Tyxal, statu ok mais action impossible

### ISSUE THREAD

**Issue by Wattrider** [fmartinou/tydom2mqtt#65]
Bonjour, j'utilise HA et cette lib avec mes équipements Delta Dore, tout fonctionne, modules de porte de garage et portail, lumiere extérieures, volets, mais l'alarme j'ai bien le statut, mais je ne parviens pas a la piloter.

je vois bien la commande partir, mais peu etre est-elle mal formatée? le code PIN de l'alarme est bon.

2022-12-19 15:43:07,036 - tydomConnector - INFO - Sending to tydom client..... PUT cdata {"value":"ON","pwd":"1234"}
2022-12-19 15:43:07,043 - tydomMessagehandler - INFO - Incoming data parsed successfully !

Merci pour votre aide

**Comment by Chalcear** [comment]
Bonjour,
Avez vous trouvé une solution?

**Comment by Wattrider** [comment]
Non, j'ai abandonné. je pilote mon alarme avec un sms en utilisant le transmetteur gsm...

**Comment by Meije38** [comment]
Bonjour, 
J'ai exactement le même problème non résolu et vous ?

**Comment by Wattrider** [comment]
non, j'abandonne Delte dore... ILs n'ont qu'a ouvrir leur protocol comme somfy!

**Comment by Meije38** [comment]
Ok , pareil j'ai aussi abandonné deltadore  je suis passé en tout zigbee mais j'ai gardé l'alarme que je pilote avec la télécommande sur laquelle j'ai soudé un relais zigbee

**Comment by Wattrider** [comment]
malin ca! moi j'envoi un sms au transmetteur gsm avec un script, on se débrouille comme on peu!

**Comment by jerry34ha** [comment]
Salut, j'ai la version 3.4.2 de mqtt2tydom et je peux contrôler la mise en route et l'arrêt de l'alarme. (je n'ai pas de zone, juste on/off).

**Comment by vinceshiver** [comment]
> Salut, j'ai la version 3.4.2 de mqtt2tydom et je peux contrôler la mise en route et l'arrêt de l'alarme. (je n'ai pas de zone, juste on/off).

Pourrais tu envoyer une capture d'écran des appareils qui remontent depuis tydom2mqtt ? Chez moi l'alarme est mal reconnu..
![Screenshot 2024-10-30 15 47 17](https://github.com/user-attachments/assets/a8873665-7676-4dc4-89ec-69569115aec6)

**Comment by vinceshiver** [comment]
Finalement j'ai réussi ! Fallait simplement ajouter le code de l'alarme dans la config de tyrdom2mqtt
